themes/bootstrap3/js/lightbox.js

```diff
@@ -89,7 +98,14 @@ VuFind.lightbox = (function() {
           || obj.url.match(/catalogLogin/))    // catalog login for holds
         ) {
           if (_originalUrl.match(/UserLogin/) || obj.url.match(/catalogLogin/)) {
-            window.location.reload();
+            var event = new CustomEvent('VuFind.lightbox.login', {
+                bubbles: true,
+                cancelable: true
+            });

```

Comment:
I don't it's possible to cancel closing the lightbox, so in that case cancelable doesn't make sense. Bubbling is not essential either since the events are fired on document, but at least they'll bubble to window too.